httpie/core.py

```diff
@@ -98,22 +101,30 @@ def program(args, env, log_error):
 
         final_response = get_response(args, config_dir=env.config.directory)
         if args.all:
+            has_branched('program', 2)
             responses = final_response.history + [final_response]
         else:
+            has_branched('program', 3)
             responses = [final_response]
 
         for response in responses:
-
             if args.check_status or downloader:
+                if args.check_status:
+                    has_branched('program', 4)
+                else:
+                    has_branched('program', 5)
```

Comment:
This only implicitly checks if either ``args.check_status`` or ``downloader`` is true. The outer most ``if`` can be entered when both are true, but in this case only ID 4 would be written. So it should be two separate ``if:s``.